ResetButton.tscn

```diff
@@ -0,0 +1,16 @@
+[gd_scene load_steps=2 format=2]
```

Comment:
What do you think of removing this scene and simply adding a `Button` node as a child of `ControlPanel`?